forum/views.py

```diff
@@ -335,8 +336,12 @@ def moderate_posts(request):
             elif action == "Delete User":
                 try:
                     post.author.delete()
+                    messages.add_message(request, messages.INFO, 'The user has been successfully deleted.')
                 except: #someone deleted him already
                     pass
+            elif action == "Delete Post":
+                messages.add_message(request, messages.INFO, 'The post has been successfully deleted.')
```

Comment:
I'll re-order these two lines not to add the message if deleting post breaks for some reason :)